.github/workflows/integration-tests.yml

```diff
@@ -383,6 +383,10 @@ jobs:
                 image: ghcr.io/balhar-jakub/cloud-gateway-service:${{ github.run_id }}-${{ github.run_number }}
             discoverable-client:
                 image: ghcr.io/balhar-jakub/discoverable-client:${{ github.run_id }}-${{ github.run_number }}
+            api-catalog-services:
```

Comment:
Because all HealthEndpointProtectionDisabledTest are included CITestsZaas. Catalog service did not have any HealthEndpointProtectionDisabledTest before, Since we added one for catalog service now, It had to be added here.